### ISSUE TITLE
Navigate between screens with Compose: Android Basics with Compose

### ISSUE THREAD

**Issue by chakshuharsh** [google-developer-training/basic-android-kotlin-compose-training-cupcake#68]
**URL of codelab**
https://developer.android.com/codelabs/basic-android-kotlin-compose-navigation?continue=https%3A%2F%2Fdeveloper.android.com%2Fcourses%2Fpathways%2Fandroid-basics-compose-unit-4-pathway-2%23codelab-https%3A%2F%2Fdeveloper.android.com%2Fcodelabs%2Fbasic-android-kotlin-compose-navigation#5

**In which task and step of the codelab can this issue be found?**
In the 6th task "Navigation to another app"

**Describe the problem**
In summaryScreen 
the onSendButtonClicked is not called using onClick function in SummaryScreen in this task of above code lab hence as showed in final image of task Link->https://developer.android.com/static/codelabs/basic-android-kotlin-compose-navigation/img/945a9df6ad9bc234_960.png
We can See "Flavor" and "Quantity" but with given steps explained in this part of code lab we cannot see this
In short results in given image are not reachable by steps explained in this art of code lab

**Steps to reproduce?**
1. Go to...
2. Click on...
3. See error...

**Versions**
_Android Studio version:_ Android Studio Giraffe | 2022.3.1
_API version of the emulator:_ 27

**Additional information**
_Include screenshots if they would be useful in clarifying the problem._
